ledger-core/runner/requestresult/outgoingresult.go

```diff
@@ -5,7 +5,22 @@
 
 package requestresult
 
+var EmptyOutgoingExecutionResult = OutgoingExecutionResult{}
```

Comment:
what is it for?

Reply:
Singleton to identify uninitialized result